usb/host/host_cdc_msc_hid/hid_app.c

```diff
@@ -99,17 +99,26 @@ void tuh_hid_umount_cb(uint8_t dev_addr, uint8_t instance)
 void tuh_hid_report_received_cb(uint8_t dev_addr, uint8_t instance, uint8_t const* report, uint16_t len)
 {
   uint8_t const itf_protocol = tuh_hid_interface_protocol(dev_addr, instance);
+  uint8_t const rpt_count = tuh_hid_instance_count(dev_addr);
```

Comment:
_Tiny_ nitpick, but perhaps `report_count` would be a better variable name? When I first read this I thought that `rpt_count` meant "repeat count" :wink:

Reply:
Done.